src/core-apps.js

```diff
@@ -0,0 +1,37 @@
+export const coreApps = [
```

Comment:
We should also hardcode the App Hub ID of each app so that matching of core apps to their App Hub version can be done without comparing names/organisations.